<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformWidget</class>
 <widget class="QWidget" name="TransformWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="from_sql_func_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>107</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>From SQL:</string>
   </property>
  </widget>
  <widget class="QLabel" name="to_sql_func_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>77</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>To SQL:</string>
   </property>
  </widget>
  <widget class="QLabel" name="language_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>107</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Language:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
